### PR TITLE
OD-611 Update step 06 to use item_tags join table

### PR DESCRIPTION
**Pull request by ariana-paris**
Mainly involves using the item_tags join table, though this also includes some refactoring on the sql usage as that was a mess.

### CODE REVIEW

File: shared_python/Tags.py [hunk 1]
```diff
@@ -30,12 +31,15 @@ def __init__(self, args, db, log):
 
 
   def create_tags_table(self, database = None):
+    """
+    Used only in step 02 for non-eFiction archives
+    """
     try:
       database = self.database if database is None else database
-      self.cursor.execute("DROP TABLE IF EXISTS {0}.`tags`".format(database))
+      self.sql.execute("DROP TABLE IF EXISTS {0}.`tags`".format(database))
```
Comment (hlieberman) [inline shared_python/Tags.py:39]: Does this work? `sql` here would be the SQL object itself; I think it should be `self.sql.cursor.execute()`.

**Review by hlieberman — CHANGES_REQUESTED** [review]
Looks good after you check the `self.sql` vs `self.sql.cursor` stuff.

File: shared_python/Tags.py (hunk 1 above)
Comment (hlieberman) [reply shared_python/Tags.py:39]: JOKES I can't read this is fine

**Review by hlieberman — APPROVED** [review]

File: shared_python/Tags.py (hunk 1 above)
Comment (ariana-paris) [reply shared_python/Tags.py:39]: I must admit that calling the custom method `execute` was not my smartest move! Maybe we can do some more refactoring once this codebase is working all the way through

**Pull request merged**